partner_shipper/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+* For the future, system can set delivery zones based on partner zip.
```

Comment:
Do we have this ROADMAP for our module too? Or it just come with what we have copied over.